vision/dcgan_mnist/dcgan_mnist.jl

```diff
@@ -99,31 +94,29 @@ function train(; kws...)
     hparams = HyperParams(; kws...)
 
     if CUDA.has_cuda()
```

Comment:
I guess this should also really change to [CUDA.functional](https://cuda.juliagpu.org/stable/api/essentials/#CUDA.functional-Tuple{Bool}):
```suggestion
    if CUDA.functional()
```